packages/widgets/src/dns-hole/controls/component.tsx

```diff
@@ -90,7 +91,17 @@ export default function DnsHoleControlsWidget({ options, integrationIds }: Widge
           </Tooltip>
 
           <Tooltip label={t("widget.dnsHoleControls.controls.setTimer")}>
-            <Button onClick={open} disabled={allDisabled} variant="light" color="yellow" fullWidth h="2rem">
+            <Button
+              onClick={() => {
+                setSelectedIntegrationIds(integrationIds);
```

Comment:
We shouldn't disable integrations that are already disabled.
Oldmarr has a neet function that filters the integrations between the 2 subset.

Also, you did something that was quite nice in oldmarr, instead of setting the list of integrations, you just by default assumed it was all disabled integrations if a specific integration ID wasn't given. But that's not important, the first part matters more.